src/components/Input/Input.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import Cleave from 'cleave.js/react';
+import * as InputMasks from './InputMasks';
+import './Input.scss';
+
+const Input = ({
+  autoComplete,
+  autoFocus,
+  className,
+  id,
+  inputMask,
+  isDisabled,
+  isRequired,
+  label,
+  name,
+  onBlur,
+  onChange,
+  onFocus,
+  placeholder,
+  type,
+  value,
+}) => {
+
+  const handleChange = (e) => {
+    onChange(e);
+  };
+
+  const handleFocus = (e) => {
+    if (onFocus) onFocus(e);
+    e.currentTarget.select();
+  };
+
+  const handleBlur = (e) => {
+    if (onBlur) onBlur(e);
+  };
+
+  const inputClasses = classNames(
+    className,
+    'input',
+    {
+      // [`font-size-${size}`]: size,
+      // [`font-color-${color}`]: color,
+    }
```

Comment:
Missing trailing comma according to the linter. I'm attempting to get use to the idea of having to add this, but can go either way (leaning towards not having a hanging comma) on whether or not we want to enforce this.